kernel/kernel-api/src/main/java/io/delta/kernel/internal/TableImpl.java

```diff
@@ -23,26 +23,42 @@
 import io.delta.kernel.exceptions.TableNotFoundException;
 import io.delta.kernel.internal.fs.Path;
 import io.delta.kernel.internal.snapshot.SnapshotManager;
+import io.delta.kernel.internal.util.Clock;
 
 public class TableImpl implements Table {
     public static Table forPath(Engine engine, String path) {
+        return forPath(engine, path, System::currentTimeMillis);
+    }
+
+    /**
+     * Instantiate a table object for the Delta Lake table at the given path.
```

Comment:
Add a comment is saying, it takes an additional parameter called {@link Clock} which helps
in testing.